treesapp_R.Rmd

```diff
@@ -12,13 +12,13 @@ You will also be provided with examples for plotting TreeSAPP classification dat
 
 ## Resources for analysis of TreeSAPP output in R
```

Comment:
The bubble plot for functional data below requires a "Function" column, but none of the required tables have this.
That cell currently fails to run